internal/pkg/service/stream/definition/repository/sink/sink_enable_test.go

```diff
@@ -259,4 +280,30 @@ func TestSinkRepository_EnableSinksOnSourceEnable(t *testing.T) {
 		require.NoError(t, err)
 		assert.True(t, sink3.IsEnabled())
 	}
+
+	// Rollback Source to the disabled state
+	// -----------------------------------------------------------------------------------------------------------------
+	{
+		now = now.Add(time.Hour)
+		source, err := d.DefinitionRepository().Source().RollbackVersion(sourceKey, now, by, 2).Do(ctx).ResultOrErr()
```

Comment:
The version 1 is when we create the `Source`? Then version 2 is 1 change performed on previously created `Source` for example.